src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -360,7 +360,7 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
       )
 
       // Redirect to status page after successful order submission
-      router.push(`/orders/${orderData.internalID}/status`)
+      window.location.reload()
```

Comment:
This is ok! I think the underlying issue is that apple pay has some kind of server-side behavior that exists outside of the MP fetch req/response cycle? 

We (should) always be super skeptical about hard reloads like this however, since as soon as the page reloads we loose overall application state aggregated up until this point. (Ideally) we fix / better understand the underlying issue and address that. Hard reloads do impact UX too, because it flashes the page.

Reply:
Cool that's what i figured 👍  For this wild flow no need to get too particular

Reply:
Correct, apple pay uses the new `Order` type, legacy checkout uses the stitched `CommerceOrder` field which is an entirely different relay object. We talked about refetching it on exit but decided that a blank slate from a reload might be preferable for the short term (since the legacy shipping step already does some actions of its own on load to get the order into an acceptable state).